README.md

```diff
@@ -231,6 +235,11 @@ commercetools/commercetools-project-sync:3.8.0 -s all
    ```bash
    docker run commercetools/commercetools-project-sync:3.8.0 -s customers
    ```
+  
+- To run the shoppingList sync
+   ```bash
+   docker run commercetools/commercetools-project-sync:3.8.0 -s shoppingLists
```

Comment:
@praveenkumarct you might need to change the increment the minor version, so not 3.8.0 but 3.9.0 for all versions, right ?

Reply:
Yes, I will update this. And build.gradle I don't need any changes ?